### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by Sliim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@ Backup Cookbook
 ===================
 [![Gitter](https://badges.gitter.im/Join Chat.svg)](https://gitter.im/damm/backup?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-This cookbook automates deploying the [backup](https://github.com/meskyanichi/backup) gem and the configuration of any *models* you may want.  With a little work you can backup everything using this cookbook as the framework.
+This cookbook automates deploying the [backup](https://github.com/backup/backup) gem and the configuration of any *models* you may want.  With a little work you can backup everything using this cookbook as the framework.
```
Comment (damm) [inline README.md]: Actually it should be https://github.com/backup/backup

File: README.md [hunk 2]
```diff
@@ -114,14 +114,12 @@ Actions:
...
     <td>Path where backup and it's configuration files and models reside</td>
     <td><tt>/opt/backup</tt></td>
-    <td></td>
   </tr>
   <tr>
     <td><tt>gem_bin_dir</tt></td>
     <td>String</td>
     <td>Path where gem binaries, such as backup, reside (e.g. "/usr/local/bin" )</td>
     <td><tt>nil</tt></td>
-    <td></td>
```
Comment (damm) [inline README.md]: I've been fond of these spaces as it looks odd without them; don't you agree?

**Comment by damm** [comment]
Other than the comments looks great; I can handle those bits if you prefer?

**Comment by Sliim** [comment]
Yes you can adapt it! ;) 
The html part was to fix malformated table, some rows had 5 columns instead of 4.
Thanks.

**Comment by damm** [comment]
Thanks you.

**Pull request merged**